support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -37,10 +37,16 @@ export type ProductDescription = {
 		{
 			billingPeriod: 'Annual' | 'Monthly' | 'Quarterly';
 			label?: string;
+			hideSimilarProductsConsent?: boolean;
```

Comment:
I have defined this as hide products and then negated it below so that it only needs to be added to three rate plans, rather than adding it to every rate plan in all the other products.